src/Symfony/Component/Translation/Tests/Extractor/PhpExtractorTest.php

```diff
@@ -88,9 +92,9 @@ public function resourcesProvider()
         return [
             [$directory],
             [$phpFile],
-            [glob($directory.'*')],
+            [glob($directory.\DIRECTORY_SEPARATOR.'*')],
```

Comment:
I suppose that this change is not needed anymore, right?

Reply:
Correct.